Tests/IndexStoreDBTests/XCTestManifests.swift

```diff
@@ -22,9 +22,14 @@ extension IndexTests {
         ("testDelegate", testDelegate),
         ("testEditsSimple", testEditsSimple),
         ("testExplicitOutputUnits", testExplicitOutputUnits),
+        ("testFilesIncludes", testFilesIncludes),
         ("testMainFilesContainingFile", testMainFilesContainingFile),
         ("testMixedLangTarget", testMixedLangTarget),
+        ("testOutOfDateEvent", testOutOfDateEvent),
+        ("testProperties", testProperties),
         ("testSwiftModules", testSwiftModules),
+        ("testSymbolsInFileC", testSymbolsInFileC),
+        ("testSymbolsInFileSwift", testSymbolsInFileSwift),
```

Comment:
Guess we were a little behind 😅

Reply:
Some of these are probably mine, sorry, didn't know it was a thing 😅